ktor-io/jvm/src/io/ktor/utils/io/charsets/CharsetJVM.kt

```diff
@@ -63,7 +63,7 @@ public actual fun CharsetEncoder.encodeUTF8(input: ByteReadPacket, dst: Output)
 
     try {
         tmp.writeDirect(0) { tmpBb ->
-            val cb = tmpBb.asCharBuffer()!!
+            val cb = tmpBb.asCharBuffer()
```

Comment:
This actually makes difference as removing it causes compiled to add assert to every every variable access. This is why I do like this to make this check only once.

Reply:
It marked with `@NotNull`, there is no platform types any more